Lib/gftools/packager.py

```diff
@@ -914,7 +920,12 @@ def _create_or_update_metadata_pb(upstream_conf: YAML,
   for font in metadata.fonts:
     font.name = upstream_conf['name']
   metadata.designer = upstream_conf['designer']
-  metadata.category = upstream_conf['category']
+
+  while metadata.category:
+    metadata.category.pop()
+  for cat in upstream_conf['category']:
+    metadata.category.append(cat)
```

Comment:
Actually, from the insight of my comment before, this is sufficient to clear the field and to assign the new values:

```py
metadata.category[:] = upstream_conf['category']
```